ckanext/dcat/profiles.py

```diff
@@ -1443,7 +1442,7 @@ def _basic_fields_graph(self, dataset_ref, dataset_dict):
         self._add_date_triples_from_dict(dataset_dict, dataset_ref, items)
 
         # Dataset URL
-        dataset_url = url_for('dataset_read',
+        dataset_url = url_for('dataset.read',
```

Comment:
We might need to check the CKAN version and use `dataset_read` on CKAN < 2.9 here

Reply:
Fixed with the new `urls.py` file